stdlib/cmake/modules/SwiftSource.cmake

```diff
@@ -928,9 +928,9 @@ function(_compile_swift_files
   endif()
 
   set(custom_env "PYTHONIOENCODING=UTF8")
-  if(SWIFTFILE_IS_STDLIB OR
+  if(SWIFT_INCLUDE_TOOLS AND (SWIFTFILE_IS_STDLIB OR
```

Comment:
@rintaro, building a standalone stdlib on linux is currently broken without this change, please review.